admin/common_auth/admin.py

```diff
@@ -39,22 +39,14 @@ def send_email_invitation(self, request, queryset):
             reset_form = PasswordResetForm({'email': user.email}, request.POST)
             assert reset_form.is_valid()
             reset_form.save(
-                #subject_template_name='templates/emails/account_creation_subject.txt',
-                #email_template_name='templates/emails/invitation_email.html',
+                subject_template_name='common_auth/emails/account_creation_subject.txt',
```

Comment:
Neither of these appear to be there. Looks like the emails directory doesn't actually exist in common_auth unless I'm looking in the wrong root directory.